packages/protocol/contracts/common/UsingPrecompiles.sol

```diff
@@ -141,10 +145,18 @@ contract UsingPrecompiles {
     return success;
   }
 
+  function getEpoch(uint256 blockNumber) public view returns (uint256) {
```

Comment:
Is this a dupe of `getEpochNumber`?

Reply:
Well `getEpochNumber` returns the current epoch, perhaps I'll change it so that they share code.